snuba/datasets/events.py

```diff
@@ -20,24 +17,14 @@
 from snuba.datasets.table_storage import TableWriter, KafkaStreamLoader
 from snuba.datasets.events_processor import EventsProcessor
 from snuba.datasets.schemas.tables import ReplacingMergeTreeSchema
+from snuba.datasets.tags_dataset import TagColumnProcessor
```

Comment:
Would it make sense to call the module `tags_processor`? It doesn't look like a dataset.

Reply:
Sure, makes sense